client/my-sites/stats/stats-date-control/stats-date-control-picker-date.tsx

```diff
@@ -10,19 +10,22 @@ const DateControlPickerDate = ( {
 	onApply,
 	onCancel,
 }: DateControlPickerDateProps ) => {
+	// TODO: Rename component?
+	// Feels a bit confusing now. Should have a better idea
+	// of appropriate names once hierarchy is finalized.
```

Comment:
I think renaming the component is a good idea! But also good to hold off a little until our hierarchy is locked down and makes more sense 👍